File: Java&Kotlin lecture/src/main/java/com/java/guess/Game.java
package com.java.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        int secret = new Random().nextInt(10) + 1;

        Scanner scanner = new Scanner(System.in);

        int number = 0;

        while (number != secret) {
            System.out.println("Please enter a number:");
            number = scanner.nextInt();

            if(number > secret) {
                System.out.println("Lower");
            } else if(number < secret) {
                System.out.println("Higher");
            } else {
                System.out.println("Bingo, number is " + number);
            }
        }
    }
}

File: Java&Kotlin lecture/src/main/kotlin/com/kotlin/guess/Game.kt
package com.kotlin.guess

import java.util.*

fun main(args: Array<String>) {
    val secret = Random().nextInt(10) + 1

    val scanner = Scanner(System.`in`)

    var number = 0

    while (number != secret) {
        println("Please enter a number:")
        number = scanner.nextInt()

        when {
            number > secret -> {
                println("Lower")
            }
            number < secret -> {
                println("Higher")
            }
            else -> {
                println("Bingo, number is $number")
            }
        }
    }
}